tests/modules/nf-scil/registration/antsapplytransforms/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+
```

Comment:
add the task.ext that you put in your main.nf.

see: https://github.com/scilus/nf-scil/pull/86/files#diff-49f7965fe10e805580b19b052c45d574eabfef118304864815b9c316a093f4dd